net/sshuttle/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2022 Thomas Cujé
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sshuttle
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=$(PKG_NAME)
+PKG_HASH:=f5a3ed1e5ab1213c7a6df860af41f1a903ab2cafbfef71f371acdcff21e69ee6
+
+PKG_MAINTAINER:=Thomas Cujé <dev.cuje@freenet.de>
+PKG_LICENSE:=LGPL-2.1-only
+
+include ../../lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+define Package/sshuttle
+  SUBMENU:=SSH
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=sshuttle: where transparent proxy meets VPN meets ssh
```

Comment:
Isn't this too long for ``make menuconfig``?

Reply:
I think it's ok
![grafik](https://user-images.githubusercontent.com/4201214/209674528-4c8b2162-3c40-4e59-8699-94b8147b9fc6.png)